tests/integration/test_azuresql.py

```diff
@@ -44,6 +49,13 @@ def test_create_table(azure_sql):
     assert table_object_id is not None
 
 
+def test_create_table_delete(azure_sql):
+    result = azure_sql.create_table(schema=SCHEMA, table=TABLE, if_exists="delete")
+    assert result == True
+    table_object_id = azure_sql.run(f"SELECT OBJECT_ID('{SCHEMA}.{TABLE}', 'U')")[0][0]
+    assert table_object_id is not None
```

Comment:
This doesn't test what it's supposed to test. 

One way to test if the `delete` option works would be to have a table with some data, run the `if_exists="delete"`, and then verify that:
a) the table's creation date remained the same (sth like this is probably available in table metadata)
b) the data has been deleted